### PR TITLE
Update protocol preview style [SCI-3416]

### PR DESCRIPTION
**Pull request by aignatov-bio**
Jira ticket: [SCI-3416](https://biosistemika.atlassian.net/browse/SCI-3416)

### What was done
Update protocol preview style. Reuse main step logic.

### CODE REVIEW

File: app/assets/stylesheets/protocols/protocol.scss [hunk 1]
```diff
@@ -0,0 +1,32 @@
+// scss-lint:disable SelectorDepth
+// scss-lint:disable NestingDepth
+
+@import "constants";
+@import "mixins";
+
+.protocol-info {
+
+  .well {
```
Comment (jbargu) [inline app/assets/stylesheets/protocols/protocol.scss]: I'm not sure why this styles cannot be used from the `protocols/index.scss'. They seem pretty similar, at least this `.well` is exactly the same. Could this be somehow DRYed up so it does not appear on 2 different places?

File: app/views/protocols/index/_protocol_preview_modal_body.html.erb [hunk 2]
```diff
@@ -68,88 +64,22 @@
...
+<%= javascript_include_tag "numeral" %>
+<%= javascript_include_tag "numeric" %>
+<%= javascript_include_tag "md5" %>
+<%= javascript_include_tag "jstat" %>
+<%= javascript_include_tag "formula" %>
+<%= javascript_include_tag "parser" %>
+<%= javascript_include_tag "ruleJS" %>
+<%= javascript_include_tag "handsontable.formula" %>
+<%= javascript_include_tag "big.min" %>
+<%= stylesheet_link_tag "handsontable.formula" %>
```
Comment (jbargu) [inline app/views/protocols/index/_protocol_preview_modal_body.html.erb]: Would it make sense to DRY up this as well (something like `_formuls_dependencies.html.erb`? It appears on couple of places as well.

File: app/assets/stylesheets/protocols/protocol.scss (hunk 1 above)
Comment (aignatov-bio) [reply app/assets/stylesheets/protocols/protocol.scss]: DRYed

File: app/views/protocols/index/_protocol_preview_modal_body.html.erb (hunk 2 above)
Comment (aignatov-bio) [reply app/views/protocols/index/_protocol_preview_modal_body.html.erb]: DRYed

**Pull request merged**